sBTC SIP

```diff
@@ -0,0 +1,166 @@
+
+sBTC will have a crucial role in scaling Bitcoin, as well as introducing new and innovative functionalities to users, growing the size of the overall Bitcoin ecosystem. This SIP aims to describe the sBTC system, the process for signer selection, and the features available in the initial release compared to subsequent versions of sBTC. It does not attempt to describe the low-level technical details of any subsequent release, which will be provided in a future SIP.
+
+## Introduction
+
+## Glossary
+
+| Term                     | Definition |
+|--------------------------|------------|
+| SIP-10 Token             | A token on the Stacks blockchain that adheres to the fungible token standards outlined in SIP-10. |
+| sBTC                     | A SIP-10 token on the Stacks Blockchain that can be turned back into BTC on the Bitcoin Blockchain. 1 sBTC is equivalent to 1 BTC on the Bitcoin Blockchain. |
+| sBTC operation           | An operation that initiates some action from the sBTC protocol. |
+| Bitcoin holder           | A person or system holding BTC, interested in acquiring sBTC. |
+| sBTC holder              | A person or system holding sBTC, interested in turning it back into BTC. |
+| .sbtc contract           | A smart contract (or a collection of contracts) defining the sBTC token and functions related to it. |
+| sBTC UTxO                | The single UTxO holding the entire BTC balance that’s pegged into sBTC. This UTxO is managed and maintained by the sBTC Signers. |
+| Stacks Signer            | The standard Stacks Signer introduced in Nakamoto. |
+| sBTC Bootstrap Signer    | A completely separate signer from the Stacks signer that will sign sBTC operations and communicate with contracts on the chain to make that feasible. This entity is registered as an sBTC signer and has partial access to spending the sBTC UTxO. This only exists in sBTC-v1. In subsequent sBTC releases this functionality will be native in the Stacks Signer. |
+| sBTC Signer              | sBTC Signer is a more general term for the entity that signs sBTC transactions under any version of sBTC. Under sBTC-v1 the sBTC signer is the sBTC Bootstrap Signer which is wholly separate from the Stacks Signer, but in subsequent releases of sBTC the Stacks Signer will also be responsible for signing sBTC operations, so it would be the sBTC Signer. |
+| sBTC Bootstrap Signer Set| The set of all sBTC signers. Each is registered with the .sbtc contract and the transfer. These entities as a group have full democratic access to the sBTC UTxO. |
```

Comment:
Is there something special about the word "bootstrap" here?  Because, this definition sounds more applicable to "sBTC signer set".